dnfdaemon-client/commands/command.cpp

```diff
@@ -25,23 +25,105 @@ along with dnfdaemon-client.  If not, see <https://www.gnu.org/licenses/>.
 #include <dnfdaemon-server/dbus.hpp>
 #include <libdnf/transaction/transaction_item_action.hpp>
 
+#include <algorithm>
 #include <iostream>
 #include <vector>
 
+#include "libdnf-cli/output/transaction_table.hpp"
+
 namespace dnfdaemon::client {
 
-void TransactionCommand::print_transaction(std::vector<dnfdaemon::DbusTransactionItem> transaction) {
-    // TODO(mblaha): print decent transaction table using smartcols
-    for (auto & ti : transaction) {
-        libdnf::transaction::TransactionItemAction action =
-            static_cast<libdnf::transaction::TransactionItemAction>(std::get<0>(ti));
-        std::cout << "Action: " << libdnf::transaction::TransactionItemAction_get_name(action) << std::endl;
-        std::cout << "Name: " << std::get<1>(ti) << std::endl;
-        std::cout << "Arch: " << std::get<5>(ti) << std::endl;
-        std::cout << "EVR: " << std::get<2>(ti) << ":" << std::get<3>(ti) << "-" << std::get<4>(ti) << std::endl;
-        std::cout << "Repo: " << std::get<6>(ti) << std::endl;
+class DbusPackageWrapper {
```

Comment:
We have another package wrapper in `commands/repoquery/repoquery.cpp:PackageDbus`. Would it be possible to leverage it instead of introducing another one? BTW I like your `DbusPackageWrapper` class name more than `PackageDbus`.

Reply:
Definitely a good point. I'll take a look

Reply:
should we file a Jira for this as well?